bench/src/bulk.rs

```diff
@@ -266,4 +284,7 @@ struct Opt {
     /// Show connection stats the at the end of the benchmark
     #[structopt(long = "stats")]
     stats: bool,
+    /// Whether to use the unordered read API
+    #[structopt(long = "unordered")]
+    read_unordered: bool,
```

Comment:
I think having the semantics flipped here (`read_ordered`, with structopt setting `true` as the explicit default) is a little easier to reason about. I would also probably just name it ordered.

Reply:
that doesn't work for boolean flags. `--something` actives them. There is no `--something false`.

Reply:
Ah, sorry about that.